tests/_support/AcceptanceTester.php

```diff
@@ -321,4 +321,20 @@ public function iConfirmTo($message)
         $this->seeInPopup($message);
         $this->acceptPopup();
     }
-}
+
+    /**
+     * @Then I should see an option :option
+     */
+    public function iShouldSeeAnOption($option)
+    {
+        $this->seeElement('input', ['type' => "button", 'value' => $option]);
+    }
+
+    /**
+     * @Then I should not see an option :option
+     */
+    public function iShouldNotSeeAnOption($option)
```

Comment:
Again, would it be better to rename this function to reflect that it is checking that an input button element is not being displayed?